test/torch/pointers/test_pointer_tensor.py

```diff
@@ -289,7 +289,7 @@ def test_move(workers):
 
     x.move(alice)
 
-    assert x.id_at_location not in bob._objects
+    assert x.id_at_location in bob._objects
```

Comment:
Having read through the code above, I think I see what you're trying to do now, which is 👍 

I still find this change in the semantics of `move` surprising though. It no longer fits my intuition of what `move` means. Should we change the method name? Introduce an additional method so we can keep the original semantics of `move`? Not sure how to handle this, but I'd be very surprised if I `moved` something and it made a copy in a new location.

Reply:
I wonder if we should call the new behavior something like `copy_to`. 🤔

Reply:
`copy_to` is legit indeed